BucketeerTests/EventInteractorTests.swift

```diff
@@ -17,10 +20,11 @@ final class EventInteractorTests: XCTestCase {
             eventDao: dao,
             clock: clock,
             idGenerator: idGenerator,
-            logger: logger
+            logger: logger,
+            featureTag: config.featureTag
         )
     }
-
+    
```

Comment:
I added one space to fix the swiftLint warning. So we could see more something like that across this PR